oracle/manifest.json

```diff
@@ -23,7 +23,9 @@
   "integration_id": "oracle",
   "assets": {
     "monitors": {},
-    "dashboards": {},
+    "dashboards": {
+      "oracle": "assets/dashboards/oracle_overview.json"
```

Comment:
Should we call the file `assets/dashboards/overview.json` for consistency ?

![image](https://user-images.githubusercontent.com/49917914/61565161-7eb37b80-aa78-11e9-836b-700a07ebdbaf.png)

Reply:
I can change this if you'd like. The benefit being that the path can be whatever as long as its specified in the manifest.json file 🎉

Reply:
Yes, I know, just nit for consistency, but feel free to dismiss my comment ;)